src/main/java/techreborn/blocks/generator/solarpanel/BlockSolarPanel.java

```diff
@@ -45,30 +56,33 @@
  * Created by modmuss50 on 25/02/2016.
  */
 public class BlockSolarPanel extends BaseTileBlock {
-
-	public static PropertyBool ACTIVE = PropertyBool.create("active");
```

Comment:
I feel like the machines should still have the on/off state so you can see that easily from the textures.